src/framework/pages/tablet/SchedulePage.java

```diff
@@ -298,12 +299,71 @@ public String getNameMeetingCreatedValue(String nameMeeting) {
 	}
 
 	/**
+	 * [JC] This method move the Time Line left or right(depend of the value)
+	 * @param nameMeeting
+	 * @return
+	 */
+	public SchedulePage moveTimeLine(int value){
```

Comment:
space between ...int value )  {
add a example
i.e. if the value is 5 is move to left, -5 is move to right... something like that